app/classifier/tasks/text/index.cjsx

```diff
@@ -48,8 +48,10 @@ module.exports = React.createClass
       annotation.value isnt '' or not task.required
 
   getDefaultProps: ->
-    task: null
-    annotation: null
+    task:
+      instruction: ''
+    annotation:
+      value: null
```

Comment:
So I'm not sure if this is an issue, but I'm getting the warning `Warning: Failed prop type: Required prop `annotation.value` was not specified in `TaskNav`.` when this is passed into `TaskNav` because it's looking for that required prop, which is given, but it's `null`. I noticed this same thing with the single choice task. Not sure if it's an issue though, as the warning doesn't cause anything to crash.